paddlecloud/notebook/views.py

```diff
@@ -206,8 +206,14 @@ def create_user_namespace(username):
                         "data": {
                             "key": encoded
                         }})
+    # create docker registry secret
     registry_secret = settings.JOB_DOCKER_IMAGE.get("registry_secret", None)
-    if registry_secret and registry_secret not in secrets:
+    registry_secret_created = False
```

Comment:
```python
secret_names = [item.metadata.name for item in secretes]
if registry_secret and registry_secret in secret_names:
    ...
```
And CephFS secrete can also use the variable `secret_names`, it's a short implementation.